uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -57,20 +84,52 @@
  * @TODO remove this patch when Shoreditch theme is migrated to SSP
  */
 .chr_manage_leave_requests {
-  .btn.ui-select-toggle {
-    padding: 4px 10px !important;
-    text-transform: none !important;
+  &[data-role="admin"] {
+    .ui-select-toggle {
+      padding: 4px 10px !important;
+      text-transform: none !important;
+      box-shadow: none !important;
+    }
+
+    .form-group {
+      overflow: inherit !important;
+    }
+
+    .panel {
+      background-image: none !important;
+    }
+
+    .panel-body, .chr_vertical_tabs {
+      background-color: #f4f7f8;
+    }
+
+    input[type="search"] {
+      width: 100% !important;
+    }
+
+    .ui-select-toggle .btn-link i {
+      margin-top: 6px;
```

Comment:
![image](https://user-images.githubusercontent.com/3973243/28061473-9af9ef5a-6622-11e7-82db-61ab58b7c079.png)

The X looks bad to me without this rule.